app/build.gradle

```diff
@@ -83,6 +83,9 @@ dependencies {
     kapt("com.squareup.moshi:moshi-kotlin-codegen:1.13.0")
     implementation("com.squareup.retrofit2:converter-moshi:2.9.0")
 
+    debugImplementation "com.github.chuckerteam.chucker:library:3.5.2"
+    releaseImplementation "com.github.chuckerteam.chucker:library-no-op:3.5.2"
```

Comment:
Even if this is a no op variant of the library. You're trusting into a third party library that could log your network in a production release. Maybe I'm a bit paranoid, but this could be because I'm used to work on banking apps.

I think it's more secure to create a specific `debug` and `release` variant of the Retrofit class, isolated in specific the folders. And adding the chucker interceptor only in the debug version.

What do you think?

Reply:
There is the network inspector in the new android studio in alpha, is this lib still necessary/relevant?